source/extensions/filters/http/istio_stats/config.proto

```diff
@@ -114,9 +114,11 @@ message PluginConfig {
 
   // Metric scope rotation interval. Set to 0 to disable the metric scope rotation.
   // Defaults to 0.
+  // DEPRECATED.
```

Comment:
just for note, we need to remove all related stuffs(e.g. the env flags) from istio since it won't work anymore.

Reply:
Yeah, that's a follow up. There's a difference in configuration since this is now a per-Envoy config, so maybe a pod annotation rather than telemetry API.

Reply:
or just mentioned in custom bootstrap.